src/interfaces/user.iterfaces.ts

```diff
@@ -0,0 +1,17 @@
+export interface IloginData {
```

Comment:
File name spelling error. Should be user.interfaces.ts

Reply:
Thank you. Done